Models/data_utils/augmentations.py

```diff
@@ -139,6 +156,52 @@ def applyTransformDepth(self, image, ground_truth):
             self.augmented_data = self.adjust_shape["mask"]
             self.augmented_image = self.adjust_shape["image"]
         return self.augmented_image, self.augmented_data
+    
+    # KEYPOINTS
+    # Set data values
+    def setDataKeypoints(self, image, ground_truth):
+
+        self.image = image
+        self.augmented_image = image
+
+        self.ground_truth = ground_truth
+        self.augmented_data = ground_truth
+
+    # Apply augmentation transform
+    def applyTransformKeypoint(self, image, ground_truth):
 
+        if (self.data_type != "KEYPOINTS"):
+            raise ValueError("Please set dataset type to KEYPOINTS in intialization of class")
+        
+        self.setDataKeypoints(image, ground_truth)
+
+        # For train set
+        if (self.is_train):
+
+            # Resize, random horiztonal flip and 10-deg rotate
+            self.adjust_shape = self.transform_shape_keypoints(
+                image = self.image,
+                mask = self.ground_truth
+            )
+            
+            self.augmented_data = self.adjust_shape["mask"]
+            self.augmented_image = self.adjust_shape["image"]
+
+            # Random image augmentations
+            if (random.random() >= 0.25 and self.is_train):
+        
+                self.add_noise = self.transform_noise(image = self.augmented_image)
+                self.augmented_image = self.add_noise["image"]
+
+        # For test/val sets
+        else:
+
+            # Only resize in test/val
+            self.adjust_shape = self.transform_shape_keypoints_test(
+                image = self.image,
+                mask = self.ground_truth
+            )
+            self.augmented_data = self.adjust_shape["mask"]
+            self.augmented_image = self.adjust_shape["image"]
 
-    
\ No newline at end of file
+        return self.augmented_image, self.augmented_data
```

Comment:
Add test function to make sure the aug works as intended. Just be visual for now.